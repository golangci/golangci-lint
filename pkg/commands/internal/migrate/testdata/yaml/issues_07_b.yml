issues:
  # Only to not generate unrelated elements inside golden.
  exclude-use-default: false
  # Only to not generate unrelated elements inside golden.
  exclude-generated: strict

  exclude-dirs-use-default: true

linters:
  disable-all: true
  enable:
    - goimports
