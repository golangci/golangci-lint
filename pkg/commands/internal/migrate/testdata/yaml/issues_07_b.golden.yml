version: "2"
linters:
  default: none
  exclusions:
    paths:
      - third_party$
      - builtin$
      - examples$
formatters:
  enable:
    - goimports
  exclusions:
    paths:
      - third_party$
      - builtin$
      - examples$
